portfolio/src/main/webapp/index.html

```diff
@@ -17,20 +21,32 @@ <h2>Click here for a random fact about me:</h2>
             <div id="greeting-container"></div>
       </div>
       
-      <form>
-        <input type="text" id="text-area"/>
-        <input type="button" value="submit" onclick="addComment()"/>
-      </form>
-
-      <div id="name-container"></div>
-      
        <!--row of photos -->
-            <img class="pictures" alt="sunset" src="/images/imageOne.jpg"/>
-            <img class="pictures" alt="dandelion" src="/images/imageTwo.jpg"/>
-            <img class="pictures" alt="san francisco" src="/images/imageThree.jpg"/>
-            <img class="pictures" alt="palace of fine arts" src="/images/imageFour.jpg"/>
-            <img class="pictures" alt="brooklyn bridge" src="/images/imageFive.jpg"/>
-            <img class="pictures" alt="flower" src="/images/imageSix.jpg"/>
+       <h1 text-align="center"> My Photos </h1>
+        <img class="pictures" alt="sunset" src="/images/imageOne.jpg"/>
+        <img class="pictures" alt="dandelion" src="/images/imageTwo.jpg"/>
+        <img class="pictures" alt="san francisco" src="/images/imageThree.jpg"/>
+        <img class="pictures" alt="palace of fine arts" src="/images/imageFour.jpg"/>
+        <img class="pictures" alt="brooklyn bridge" src="/images/imageFive.jpg"/>
+        <img class="pictures" alt="flower" src="/images/imageSix.jpg"/>
+        
+        <div id="experienceSection"> 
+            <h1> My Experience </h1>
+            <div id="chart-container"></div>
+        </div>
+
+        <div id="mapSection">
+            <h1> Places Visited </h1>
+            <div id="map"></div>
+        </div>
+
+        <h2> Comments </h2>
```

Comment:
Remove the extra spaces inside the h2 tag. I'll render the same with or without the spaces.